modules/Core/pages/panel/api.php

```diff
@@ -233,7 +233,7 @@
             'API' => $language->get('admin', 'api'),
             'PAGE' => PANEL_PAGE,
             'API_INFO' => $language->get('admin', 'api_info', [
-                'pluginLinkStart' => '<a href="https://plugin.namelessmc.com" target="_blank">',
```

Comment:
I think this should stay as `plugin.namelessmc.com`

Reply:
But then, in README, should it be that?